chains/8453.json

```diff
@@ -1083,5 +1083,10 @@
         "addr": "0x08eDEbFFaE68970DCf751baa826182b3a4aCFC05",
         "isContract": true,
         "name": "ANTHIAS_MULTISIG"
+    },
+    {
+        "addr": "0x8BB4C975Ff3c250e0ceEA271728547f3802B36Fd",
```

Comment:
Created over 2 years ago by 0x9f76a95AA7535bb0893cf88A146396e00ed21A12 deployer, is this correct? Assuming so but just verifying.